main.tf

```diff
@@ -161,3 +161,43 @@ resource "aws_organizations_policy_attachment" "deny_all_access" {
   policy_id = aws_organizations_policy.deny_all_access.id
   target_id = element(var.deny_all_access_target_ids.*, count.index)
 }
+
+#
+# Require S3 encryption
+#
+
+data "aws_iam_policy_document" "require_s3_encryption" {
+  statement {
+    effect    = "Deny"
+    actions   = ["s3:PutObject"]
+    resources = ["*"]
+    condition {
+      test     = "StringNotEquals"
+      variable = "s3:x-amz-server-side-encryption"
+      values   = ["AES256"]
+    }
+  }
+  statement {
+    effect    = "Deny"
+    actions   = ["s3:PutObject"]
+    resources = ["*"]
+    condition {
+      test     = "Bool"
+      variable = "s3:x-amz-server-side-encryption"
+      values   = [false]
```

Comment:
Maybe I'm reading it wrong, but looking at AWS's example [here](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_example-scps.html#example-require-encryption) should this `false` not be `true`?

Reply:
Good eye. The test criteria there is "Null", which basically means does the key exist in the request. This makes more sense to me and I'll match what they have.

Reply:
Booleans are always hard for me often times, so I could totally be wrong here. Happy to accept this is right. Simply wanted to raise it.